test/Narato.ResponseMiddleware.Models.Test/Serialization/ExceptionSerializationTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Narato.ResponseMiddleware.Models.Exceptions;
+using Narato.ResponseMiddleware.Models.Models;
+using System;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using Xunit;
+
+namespace Narato.ResponseMiddleware.Models.Test.Serialization
+{
+    public class ExceptionSerializationTest
+    {
+        [Fact]
+        public void EntityNotFoundSerializationTest()
+        {
+            var ErrorCode = "Test";
+            var exception = new EntityNotFoundException(ErrorCode, "Test Message");
+            try {
+                var deserializedException = Clone(exception);
+                Assert.Equal(deserializedException.ErrorCode, ErrorCode);
+            }
+            catch(Exception ex) {
+                Assert.Null(ex.Message);
```

Comment:
This is a very weird way of testing... Don't do a try-catch in a test method! Either you expect an exception, and you use Assert.Throws<>()
or you don't expect an exception, and you just do what you do within the try clause now

this comment applies to every test method you wrote :)

Reply:
Good point :)